lib/getNextJobs.js

```diff
@@ -1,6 +1,28 @@
 'use strict';
 
 const { PR_JOB_NAME, EXTERNAL_TRIGGER_ALL } = require('screwdriver-data-schema').config.regex;
+const STAGE_SETUP_PATTERN = /^stage@([\w-]+)(?::setup)$/;
```

Comment:
Define this regex in data-schema so that it can be used in other modules
https://github.com/screwdriver-cd/data-schema/blob/master/config/regex.js